cGame.h

```diff
@@ -79,7 +78,7 @@ class cGame : public cScenarioObserver, cInputObserver {
 
 	int m_pathsCreated;
 
-    int m_musicVolume;              // volume of the mp3 / midi
+    int m_musicVolume;              // volume of the music
```

Comment:
This data member is not used. Shall we leave it in for now? I believe the plan is to have volume settings in the menu at some point.

Reply:
🤔 , I guess we can keep it. Once we have a proper configuration object we can move it to there.